source/efficient_mode/EfficientHost.h

```diff
@@ -20,17 +20,25 @@ class EfficientHost: public Host {
     *
   */
   emp::Ptr<EfficientWorld> my_world = NULL;
+
+  /**
+    *
+    * Purpose: Represents the configuration settings for a particular run.
```

Comment:
Let's put in a note here to be more verbose since this can be confusing. "Note: holds all configuration settings and points to same configuration object as my_config, but with the correct subtype."